src/assets/stylesheets/index.scss

```diff
@@ -673,618 +673,6 @@ section.navbar-fixed {
   overflow: hidden;
 }
 
-/* navbar
-=================================================================== */
-
-.navbar {
-  padding: 0;
-  min-height: 75px;
-  box-shadow: 0 0 3px rgba(0, 0, 0, 0.2);
-}
-.navbar-brand {
-  /* background: $logo-dark no-repeat center center / contain; */
-  width: 120px;
-  @extend .text-hide;
-  height: 75px;
-  padding: 25px 0;
-}
-.navbar-brand img {
-  margin: 0;
-}
-
-.nav {
-  margin: 0;
-}
-.nav.navbar-right {
-  margin-left: auto;
-  margin-right: 45px;
-}
-
-.nav > li,
-.nav > li > a {
-  position: relative;
-  display: block;
-}
-.navbar-nav > li > a {
-  height: 75px;
-  position: relative;
-  padding: 25px 15px !important;
-  text-transform: uppercase;
-  font-weight: 600;
-  font-size: 14px;
-  color: $theme-greydark;
-  display: inline-block;
-}
-
-.nav > li:hover > a,
-.nav > li:focus > a {
-  background: transparent;
-}
-.nav > li.open > a,
-.nav > li.open > .menu-toggle {
-  background: $nav-hover !important;
-  color: #fff !important;
-}
-
-.nav li.active > a,
-.nav li.active > .menu-toggle {
-  background: $primary !important;
-  color: #fff !important;
-}
-
-.nav > li > .dropdown-menu {
-  border-radius: 0;
-  border: none;
-  left: 50%;
-  margin-top: 0px;
-  right: auto;
-  text-align: left;
-  position: absolute !important;
-  transform: translate(-50%, 20px);
-  box-shadow: 0 0 8px rgba(0, 0, 0, 0.1), 0 -20px 10px rgba(44, 44, 44, 0.04);
-  padding: 10px;
-  font-size: 14px;
-}
-
-.nav > li > .dropdown-menu:before {
-  content: "";
-  display: block;
-  width: 100%;
-  height: 20px;
-  position: absolute;
-  top: -20px;
-  left: 0;
-  right: 0;
-}
-.nav > li > .dropdown-menu:after {
-  bottom: 100%;
-  left: 50%;
-  border: solid transparent;
-  content: " ";
-  height: 0;
-  width: 0;
-  position: absolute;
-  pointer-events: none;
-  border-color: rgba(255, 255, 255, 0);
-  border-bottom-color: #fff;
-  border-width: 10px;
-  margin-left: -10px;
-}
-
-.nav .dropdown-menu > li > a {
-  color: $theme-grey;
-  padding: 5px 10px !important;
-  text-align: center;
-}
-
-.nav .dropdown-menu > li > a:hover,
-.nav .dropdown-menu > li > a:focus {
-  background: $nav-hover;
-  color: #fff;
-}
-.nav .dropdown-menu > li.active > a:hover,
-.nav .dropdown-menu > li.active > a:focus {
-  background: $primary;
-  color: #fff;
-}
-.open > .dropdown-menu {
-  display: block;
```

Comment:
Where did this rule go? Is it not important?

Reply:
It's here: https://github.com/Scrivito/scrivito_example_app_js/pull/229/files#diff-9e04cc0a7b0d44695b075612dc170e51R283